internal/pkg/cli/deploy/deploy.go

```diff
@@ -812,6 +905,20 @@ func (d *workloadDeployer) uploadArtifactsToS3(in *uploadArtifactsToS3Input) (up
 	}, nil
 }
 
+func (d *workloadDeployer) uploadCustomResources(workloadType string, customResources func(template.Reader) ([]*customresource.CustomResource, error)) (map[string]string, error) {
```

Comment:
do we need this parameter `workloadType`? It seems like it is only used for error msg.

Reply:
Do you have a recommendation for an alternative way of writing it? otherwise the error messages won't be specific enough

Reply:
Got rid of it!